app/assets/javascripts/discourse/app/lib/transformer/registry.js

```diff
@@ -7,6 +7,7 @@ export const VALUE_TRANSFORMERS = Object.freeze([
   // use only lowercase names
   "category-description-text",
   "category-display-name",
+  "mentions-class",
```

Comment:
Bikeshedding but... the transformer runs for each mention (singlular), and acts on an array of classes (plural). So I think it might be better as:

```
"mention-classes"
```
?